app/views/notes/index.html.erb

```diff
@@ -39,13 +39,7 @@
   <%= render :partial => "notes/notes" %>
 
   <hr />
-  
 </div>
 
-<% if params[:controller] == "tag" || params[:controller] == "search" %>
-  <%= render :partial => "sidebar/related" %>
-<% elsif params[:action] == "author" || params[:action] == "author_topic" %>
-  <%= render :partial => "sidebar/author" %>
-<% else %>
-  <%= render :partial => "sidebar/user" %>
-<% end %>
+<%# The sidebar is also used in the admin/queue section,
+which reuses the 'notes/index' template %>
```

Comment:
The sidebar is also used in the admin/queue section, which reuses the 'notes/index' template, not sure if it should be removed.